routing-daemon/conf/routing-daemon.conf

```diff
@@ -39,6 +41,10 @@ MONITOR_TYPE=http-ecv
 BIGIP_HOST=127.0.0.1
 BIGIP_USERNAME=admin
 BIGIP_PASSWORD=passwd
+#This ssh key must be added to the admin user's .ssh/authorized_keys
```

Comment:
Might be helpful to be more explicit: "the admin user's .ssh/authorized_keys on the F5 BIG-IP LTM host".